007_productOfNumbersArray.js

```diff
@@ -0,0 +1,15 @@
+const readline = require('readline-sync');
+
+const arrayMultiply = (elements) => {
+    let numberArray = elements.split(",");
+    let arrayLength = numberArray.length;
+    let product = [];
+    for (i = 0; i < arrayLength; i++) {
```

Comment:
use let here

Reply:
This file (007_productOfNumersArray.js) was wrongly inserted to this branch. this is removed and made changes in the original file (006_sumOfNumbersArray.js).
please review